src/Node/Jwt.js

```diff
@@ -1,31 +1,65 @@
-var jwt = require("jsonwebtoken");
+const jwt = require("jsonwebtoken");
 
-exports._decode = function decode(just, nothing, token) {
+const registeredClaimsKeys = ["iss", "sub", "aud", "exp", "nbf", "iat", "jti"];
+
+// TODO: Remove this when https://github.com/erikd/language-javascript/pull/118 is merged
```

Comment:
Hopefully merged soon, the partitionClaims function is unsafe, dirty, and verbose 😕